client/dashboard/domains/domain-transfer/inbound-transfer/in-progress.tsx

```diff
@@ -0,0 +1,63 @@
+import { Badge } from '@automattic/ui';
+import { Icon, __experimentalVStack as VStack } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { drafts, layout } from '@wordpress/icons';
+import { siteOverviewRoute } from '../../../app/router/sites';
+import Notice from '../../../components/notice';
+import RouterLinkSummaryButton from '../../../components/router-link-summary-button';
+import { Text } from '../../../components/text';
+import { InboundTransferStep } from './transfer-step';
+
+export const InboundTransferInProgress = ( {
+	domainName,
+	siteSlug,
+}: {
+	domainName: string;
+	siteSlug: string;
+} ) => {
+	return (
+		<InboundTransferStep
+			icon={ <Icon size={ 24 } icon={ drafts } /> }
+			title={ domainName }
+			badge={ <Badge intent="warning">{ __( 'In progress' ) }</Badge> }
+			subtitle={ __( 'Estimated: 5–7 days' ) }
+			progress={ { currentStep: 2, color: 'var(--wp-admin-theme-color)' } }
+		>
+			<VStack spacing={ 8 }>
+				<Notice>
+					{ __(
+						"Domain name transfers typically take 5–7 days. We'll email you when it's ready."
```

Comment:
Nit: [we should be using a curly apostrophe](https://github.com/Automattic/wp-calypso/blob/trunk/client/dashboard/docs/typography-and-copy.md#:~:text=Use%20curly%20quotes%20and%20apostrophes.%20e.g.%20%E2%80%9Clike%20this%E2%80%9D%20instead%20of%20%22like%20this%22%20e.g.%20it%E2%80%99s%20instead%20of%20it%27s).